src/models/env_config.cpp

```diff
@@ -18,14 +18,41 @@
 
 bool enableCATMLP() {
     static int catMlp = -1;
-    if (catMlp == -1)
-        catMlp = (getenv("ENABLE_CAT_MLP") ? atoi(getenv("ENABLE_CAT_MLP")) : 1);
+    if (catMlp == -1) catMlp = (getenv("ENABLE_CAT_MLP") ? atoi(getenv("ENABLE_CAT_MLP")) : 1);
     return catMlp == 1;
 }
 
+bool tunedComm() {
```

Comment:
it is not so easy to understand "Tuned communication". add some comment?

Reply:
Added~